presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -185,7 +185,7 @@ public void testCreateTable()
         assertQueryFails("CREATE TABLE " + tableName + " (a bad_type)", ".* Unknown type 'bad_type' for column 'a'");
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
 
-        tableName = "test_create_table_if_not_exists_" + randomTableSuffix();
+        tableName = "test_cr_tab_not_exists_" + randomTableSuffix();
```

Comment:
ouch.... This is for oracle, right?
we could use oracle 12.2 and enjoy longer table names ...

Reply:
I could not find 12.2 docker image in the wild. Let's merge this for now.